app/controllers/sales_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SalesController < ApplicationController
+  def new
+    @vendor = Vendor.find(params[:vendor_id])
+    @sale = @vendor.sales.new
+    @product = Product.where(:vendor_id => @vendor.id)
+  end
```

Comment:
This is confusing to me that @product is singular, but it's going to return multiple products. Either I'm reading the code incorrectly, or it would be helpful for the variable to be plural.